Tests/Common/Securities/SecurityTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Specialized;
+using System.Linq;
+using NUnit.Framework;
+using org.apache.avalon.framework;
+using QuantConnect.Data;
+using QuantConnect.Data.Market;
+using QuantConnect.Securities;
+using QuantConnect.Algorithm;
+using QuantConnect.Orders.Fills;
+using QuantConnect.Orders.Fees;
+using QuantConnect.Orders.Slippage;
+
+namespace QuantConnect.Tests.Common.Securities
+{
+    [TestFixture]
+    public class SecurityTests
+    {
+        [Test]
+        public void SimplePropertiesTests()
+        {
+            var exchangeHours = SecurityExchangeHours.AlwaysOpen(TimeZones.NewYork);
+            var config = CreateTradeBarConfig();
+            var security = new Security(exchangeHours, config);
+
+            Assert.AreEqual(config, security.SubscriptionDataConfig);
+            Assert.AreEqual(config.Symbol, security.Symbol);
+            Assert.AreEqual(config.SecurityType, security.Type);
+            Assert.AreEqual(config.Resolution, security.Resolution);
+            Assert.AreEqual(config.FillDataForward, security.IsFillDataForward);
+            Assert.AreEqual(exchangeHours, security.Exchange.Hours);
```

Comment:
Instead of this test, my first option was:
`Assert.AreEqual(new SecurityExchange(exchangeHours), security.Exchange);`
It didn't pass and I don't know why.